src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj

```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.5.0-preview" />
-    <ProjectReference Include="..\Management.Search\Management.Search.csproj" />
+    <ProjectReference Include="..\Management.Search\Microsoft.Azure.Management.Search.csproj" />
```

Comment:
@gucalder do you have any changes for the Search management SDK? Or was this is a mistake.